pkg/vault/contracts/Vault.sol

```diff
@@ -140,10 +140,12 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
 
     /// @inheritdoc IVaultMain
     function sendTo(IERC20 token, address to, uint256 amount) external nonReentrant onlyWhenUnlocked {
-        _takeDebt(token, amount);
-        _reservesOf[token] -= amount;
+        if (amount > 0) {
```

Comment:
I'd remove this from here and prefer avoiding the calls from the router whenever possible. Just to keep it extra simple.

`_takeDebt` already does nothing for `amount == 0` anyways.

Reply:
Ok; will try. Having it here reduces the changes to the Router. I agree it's cleaner to have it 100% in the router (though it imposes this burden on future routers), as long as it doesn't exceed the bytecode.

Reply:
I'm not sure this is better. Not checking in `sendTo` adds a lot of code to the Routers - and I can't seem to avoid having to change VaultAdmin anyway.